test/parallel/test-abortcontroller.js

```diff
@@ -1,7 +1,7 @@
-// Flags: --no-warnings --expose-gc --expose-internals
+// Flags: --expose-gc
```

Comment:
I think we should add an eslint rule to avoid re-adding `expose-internals` to files that we avoid.